app/controllers/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  after_action :verify_authorized
```

Comment:
This is redundant since `ApplicationController` already implements it.

Reply:
I didn't get what is redundant, the whole controller?